File: Equivalent mutations/original C# code/2D_to_1D_no_1D.cs
//http://www.sanfoundry.com/csharp-program-convert-2darray-1darray/

/*
 * C# Program to Convert a 2D Array into 1D Array
 */
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
 
namespace Program
{
    class twodmatrix
    {
        int m, n;
        int[,] a;
        twodmatrix(int x, int y)
        {
            m = x;
            n = y;
            a = new int[m, n];
        }
        public void readmatrix()
        {
            for (int i = 0; i < m; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    Console.WriteLine("a[{0},{1}]=", i, j);
                    a[i, j] = Convert.ToInt32(Console.ReadLine());
                }
            }
        }
        public void printd()
        {
            for (int i = 0; i < m; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    Console.Write("{0}\t", a[i, j]);
 
                }
                Console.Write("\n");
            }
        }
        public void convert()
        {
            return;
        }
        public void printoned()
        {
            for (int i = 0; i < m; i++)
            {
                for (int j = 0; j < n; j++)
                {
                    Console.WriteLine("{0}\t", a[i*n+j]);
                }
            }
        }
 
 
        public static void Main(string[] args)
        {
            twodmatrix obj = new twodmatrix(2,3);
            Console.WriteLine("Enter the Elements : ");
            obj.readmatrix();
            Console.WriteLine("\t\t Given 2-D Array(Matrix) is : ");
            obj.printd();
            Console.WriteLine("\t\t Converted 1-D Array is : ");
            obj.printoned();
            Console.ReadLine();
        }
    }
}

File: experiments/wermer/Equivalent mutations/Mutants/2D_to_1D_no_1D.java
//http://www.sanfoundry.com/csharp-program-convert-2darray-1darray/


class twodmatrix
{
    int m, n;
    int[][] a;
    int[] b;
    twodmatrix(int x, int y)
    {
        this.m = x;
        this.n = y;
        this.a = new int[this.m][this.n];
    }
    public void readmatrix()
    {
        for (int i1 = 0; i1 < this.m; i1++)
        {
            for (int j1 = 0; j1 < this.n; j1++)
            {
                System.out.printf("a[{0},{1}]=", i1, j1);
                this.a[i1][j1] = System.in.read();
            }
        }
    }
    public void printd()
    {
        for (int i2 = 0; i2 < this.m; i2++)
        {
            for (int j2 = 0; j2 < this.n; j2++)
            {
                System.out.printf("{0}\t", this.a[i2][j2]);
 
                }
                System.out.printf("\n");
        }
    }
    public void printoned()
    {
            for (int i4 = 0; i4 < this.m; i4++)
            {
                for (int j4 = 0; j4 < this.n; j4++)
                {
                    Console.WriteLine("{0}\t", a[i*n+j]);
                }
            }
    }
 
 
    public static void main(String[] args)
    {
        twodmatrix obj = new twodmatrix(2,3);
        System.out.printf("Enter the Elements : ");
	    obj.readmatrix();
	    System.out.printf("\t\t Given 2-D Array(Matrix) is : ");
	    obj.printd();
	    System.out.printf("\t\t Converted 1-D Array is : ");
	    obj.printoned();
    }
}